InOut/rtalsa.c

```diff
@@ -1856,7 +1856,7 @@ int listAlsaSeq(CSOUND *csound, CS_MIDIDEVICE *list, int isOutput) {
       /* reset query info */
       snd_seq_port_info_set_client(pinfo, snd_seq_client_info_get_client(cinfo));
       snd_seq_port_info_set_port(pinfo, -1);
-      count = 0;
+      // count = 0;
```

Comment:
Is this not read at all? Then maybe delete. Shouldn't have commented code

Reply:
Ok, deleted instead of commented unused assignments